modules/govcms_ckan_media/includes/MediaInternetGovCmsCkanHandler.inc

```diff
@@ -34,8 +34,8 @@ class MediaInternetGovCmsCkanHandler extends MediaInternetBaseHandler {
     // @baseurl/dataset/PACKAGE_ID/resource/RESOURCE_ID/download/filename.csv
     // @baseurl/dataset/PACKAGE_ID/resource/RESOURCE_ID
     $patterns = array(
-      '@' . $this->baseHost . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)/download/(?:[^"\&\? ]+)@i',
-      '@' . $this->baseHost . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)@i',
+      '@' . $this->baseUrl . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)/download/(?:[^"\&\? ]+)@i',
+      '@' . $this->baseUrl . '/dataset/([^"\&\? ]+)/resource/([^"\&\? ]+)@i',
```

Comment:
That one was changed to support ckan endpoints using different ports, could we instead change where baseUrl is set to also include the path if it exists? 

Eg. https://github.com/govCMS/govcms-ckan/blob/7.x-1.x/modules/govcms_ckan_media/includes/MediaInternetGovCmsCkanHandler.inc#L182